packages/sui-studio/src/components/navigation/_style.scss

```diff
@@ -1,6 +1,5 @@
 .sui-StudioNav {
   border-radius: 4px;
-  box-shadow: inset 4px 4px 12px #dddddd;
```

Comment:
![image](https://user-images.githubusercontent.com/1674036/202991078-79a2b2bb-aa70-4f30-9949-44dec6f09c34.png)
This line is no longer in production. Dont't know why its in the current master VCS branch